Gems/Atom/Feature/Common/Code/Source/FrameCaptureSystemComponent.cpp

```diff
@@ -908,64 +969,52 @@ namespace AZ
 
             if (!filePathA.ends_with(".png") || !filePathB.ends_with(".png"))
             {
-                AZ_Error("FrameCaptureSystemComponent", false, "Image comparison only supports png and ppm files for now.");
-                result.m_resultCode = Utils::ImageDiffResultCode::UnsupportedFormat;
-                return result;
+                error.m_errorMessage = "Image comparison only supports png and ppm files for now.";
```

Comment:
Error says ppm is supported but the if check only looks for png.

Reply:
Forgot to remove that. I wanted to support both png and ppm in the beginning. But ppm loader doesn't have equivalent functions.

Reply:
Still says ppm :)

Reply:
Oops. Should be fixed now.